src/app/content/components/popUp/ColorFilter.tsx

```diff
@@ -40,17 +57,20 @@ const ColorFilter = ({className}: Props) => {
     <AllOrNone
       onNone={() => setSelectedColors([])}
       onAll={() => setSelectedColors(Array.from(colorFiltersWithContent))}
+      disabled={disabled}
     />
-    {highlightStyles.map((style) => <Checkbox
+    {styles.map((style) => <Checkbox
       key={style.label}
       checked={selectedColorFilters.has(style.label)}
-      disabled={!colorFiltersWithContent.has(style.label)}
+      disabled={disabled || !colorFiltersWithContent.has(style.label)}
       onChange={() => handleChange(style.label)}
     >
       <ColorIndicator style={style} size='small'/>
-      <FormattedMessage id={`i18n:highlighting:colors:${style.label}`}>
-        {(msg: Element | string) => msg}
-      </FormattedMessage>
+      <ColorLabel>
```

Comment:
for styling purpose, to have a wrapper that contains only text and style it according to the design
It was behaving weird If I were styling `Checkbox`:
![image](https://user-images.githubusercontent.com/31478212/89530795-603d3100-d7ef-11ea-9147-07e473e06ac7.png)
It could be probably solved with a css grid, but I think this approach is cleaner

Reply:
that sounds fine then, what is the weirdness that this fixes?